modules/updating-eus-to-eus-upgrade-console.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing-eus-eus-upgrade.adoc
+
+:_content-type: PROCEDURE
+[id="updating-eus-to-eus-upgrade-console_{context}"]
+= EUS-to-EUS upgrade using the web console
+
+.Prerequisites
+
+* Have access to the web console as a user with `admin` privileges.
+
+.Procedure
+
+. Upgrade any OLM Operators to versions that are compatible with both versions you are updating to. You can find these steps on the link:https://docs.openshift.com/container-platform/4.11/operators/admin/olm-upgrading-operators.html#olm-changing-update-channel_olm-upgrading-operators[Preparing for an Operator update] page.
+
+. Verify that all machine config pools display a status of `UPDATED` and that no machine config pool displays a status of `UPDATING`.
++
+To view the status of all machine config pools, click *Compute* -> *MachineConfigPools* and review the contents of the *Update status* column.
++
+[NOTE]
+====
+If your machine config pools have an `Updating` status, please wait for this status to change to `Updated`. This process could take several minutes.
```

Comment:
same as above, we should `Updated` replaced with `Up to date`

Reply:
done :)

Reply:
I'm sorry, did you make the changes? I still see `Updated` here ~~ @obrown1205